packages/dashboard/web/tailwind.input.css

```
@@ -0,0 +1,450 @@
+@tailwind base;
```

Biome (2.4.4)

[error] Unexpected unknown at-rule: tailwind 

(lint/suspicious/noUnknownAtRules)